alert/post.py

```diff
@@ -21,8 +21,6 @@
 # Post detected alerts
 with open('dashboard/regressions.json') as f:
     regressions = json.load(f)
-    poster.set_server_url("http://localhost:8080")
```

Comment:
These options are already set at the beginning of the file, so I just removed the duplicate lines where we're setting them again to the same values.